src/NamespaceManager.php

```diff
@@ -60,7 +60,7 @@ public function init( $vars ) {
 		$newVars = [];
 
 		if ( !$this->isDefinedConstant( 'SMW_NS_PROPERTY' ) ) {
-			Globals::replace( $newVars = $this->initCustomNamespace( $vars ) );
+			Globals::replace( $newVars = $this->initCustomNamespace( $vars )['newVars'] );
```

Comment:
Why is initCustomNamespace returning two variables in an array?

Reply:
initCustomNamespace() originally returned $instance.
For testing it also needs to return $newVars.

So a quick solution was to return:
	'instance' => $instance,
	'newVars' => $newVars

I considered setting a class variable '$instance', then call it from 'initCanonicalNamespaces()';
however, both methods are static.